.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+<!-- If reporting a bug: -->
```

Comment:
I think Docker somehow relates to OS version, so I didn't separate that. 
Bare Metal or Cloud shouldn't really matter in most cases .
Same with client, maybe for a generic issue tracker it would be interesting - but this here should be for server related issues.